go.mod

```diff
@@ -169,6 +170,8 @@ require (
 // )
 replace cosmossdk.io/x/protocolpool => ./x/protocolpool
 
+replace cosmossdk.io/x/gov => ./x/gov
```

Comment:
I think this can be removed right? The sdk does not depend on gov anymore.
Then I think you can afterward so remove it from client v2

Reply:
go mod tidy fails without this.